src/wasm/kotlin/src/arcs/Particle.kt

```diff
@@ -133,7 +133,7 @@ open class Singleton<T : Entity<T>>(val entityCtor: () -> T) : Handle() {
 }
 
 class Collection<T : Entity<T>>(private val entityCtor: () -> T) : Handle(),
-    Iterable<T> {
+        Iterable<T> {
```

Comment:
4-space continuation indent, and both Handle and Iterable should be on the new line.

Reply:
I decided to move `Iterable` up.